src/Calypso-SystemTools-QueryBrowser-Tests/MockClyBrowser.class.st

```diff
@@ -0,0 +1,34 @@
+Class {
+	#name : #MockClyBrowser,
```

Comment:
All Calypso classes are prefixed with Cly. Why mock should be different? 
I would use name ClyBrowserMock in that case

Reply:
Ideally we should replace the class with a more general MockObject class - without necessarily going as far as including Mocketry or similar.

I'll rename it to ClyMockBrowser for now.